File: benchmark/results/python-invocation-script/run-005/a-mcp/output/notification/services/notification_service.py
from notification.aggregates.notification import Notification

"""NotificationService service."""
class NotificationService:
    def create(self, input: Notification) -> Notification:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Notification | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Notification]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/ts-multilang/run-002/b-baseline/output/src/domain/notification/services/NotificationService.ts
import { Notification } from "../aggregates/Notification.js";

/** NotificationService domain service. */
export class NotificationService {
  create(input: Partial<Notification>): Notification {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): Notification | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): Notification[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}
